jobOptions.mac

```diff
@@ -15,4 +15,17 @@
 /WCSim/physics/list WCSim # included for backward-compatibility.  Requires jobOptions2.mac
 
 
-
+#choose the Trigger & Digitizer type (and options)
+# set at least one (both to be sure) to SKI_SKDETSIM if you want to use the old combined Digitization & Trigger classes
+/DAQ/Digitizer SKI
+/DAQ/Trigger NHits
+#do we want to save only triggered events (mode 0), both triggered events & failed events (mode 1), or only failed events (mode 2)?
+/DAQ/TriggerSaveFailures/Mode 0
+#what trigger time to use for the failed events?
```

Comment:
The current Digitizer/Trigger are direct analogues of the SKI implementation of the old class. But of course, since the reordering, we're now some hybrid of SKI & SKIV electronics/DAQ that has never been built. So the default numbers have been chosen for a direct comparison to the old code. @jeimber is working on implementing real SKIV with #13, which will of course have "reality" as a major driving force.
Just to be clear, the TriggerSaveFailures options are for trigger development, and not to be seen as a 'real' trigger.